shop/src/main/java/ro/msg/learning/shop/model/BaseEntity.java

```diff
@@ -0,0 +1,14 @@
+package ro.msg.learning.shop.model;
+
+import lombok.Data;
+
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@Data
+@MappedSuperclass
+
+public class BaseEntity {
+    @Id
+    private int id;
```

Comment:
Here you should also give it the auto generation adnotation so it will automatically give itself a value (if last created id is 5 it will give the new object a 6)